zest/releaser/utils.py

```diff
@@ -695,9 +695,17 @@ def _execute_command(command, input_value=''):
     stderr_output = e.read()
     # We assume that the output from commands we're running is text.
     if not isinstance(stdout_output, six.text_type):
-        stdout_output = stdout_output.decode(OUTPUT_ENCODING)
+        try:
+            stdout_output = stdout_output.decode(OUTPUT_ENCODING)
+        except UnicodeDecodeError:
+            stdout_output = '%s stdout output is not a valid %s stream.' % (
```

Comment:
Another way to handle that would be not to decode the output.

Since we are printing it back we could just keep it as bytes instead so that the terminal whould have to handle the display and it would look really close to the original one.